scipy-1.0/paper.tex

```diff
@@ -151,6 +151,34 @@ \subsection*{Numerical optimization}
 
 
 \subsection*{Statistical distributions}
+Version 1.0 of SciPy saw the addition of the energy distance algorithm in
+\texttt{scipy.stats.energy\_distance}. Energy distance is used in a varity of applications ranging
+from genetic analysis to multivariate independence. We also added the
+first Wasserstein distance algorithm to the library in
+\texttt{scipy.stats.wasserstein\_distance}. The Wasserstein distance is often
+used in color digital image analysis.
+
+We now also provide the cumulative distribution function of the multivariate
+normal distribution, \\\texttt{scipy.stats.multivariate\_normal}, via the newly
+associated \texttt{logcdf} and \texttt{cdf} methods.
+
+Other recent additions to \texttt{scipy.stats} include the Argus distribution
+in \texttt{scipy.stats.argus} (used in particle physics), a class for
+generating a distribution from a histogram in
+\texttt{scipy.stats.rv\_histogram}, an implementation of the multinomial
+distribution in \texttt{scipy.stats.multinomial} (used for example in natural
```

Comment:
I like that you've given an example use of the multinomial distribution. It would be nice if the stats experts could include similar commentary for other new features - or it would be OK if we decide not to give examples uses at all. So I suppose the suggestion is to be consistent in the level of detail provided for these items that are listed like this unless there is a particular reason to highlight one as you have.

If this example were to stand alone, I believe "for example" deserves to be surrounded by parentheses here:
"used, for example, in natural language processing"
or maybe used like:
"used in natural language processing, for example"
https://data.grammarbook.com/blog/commas/commas-part-10/